### ISSUE TITLE
AttributeError: 'NoneType' object has no attribute 'erase_screen'

### ISSUE THREAD

**Issue by mikelococo** [shadowmoose/RedditDownloader#79]
### Describe the bug
Error and apparent hang with the message: `AttributeError: 'NoneType' object has no attribute 'erase_screen'`

### Environment Info
 - **OS:** Ubuntu 16.04.6 LTS, Python 3.5.2
 - **RMD Version:** Master at b9c7f81665ca5ce94ceacbc0e85d9dc69b6584af

### Screenshots/Information
```
$ python redditdownloader                                                                                                                                                                                                            

====================================
   Reddit Media Downloader 3.0.0
====================================
    (By ShadowMoose @ Github)

Loaded settings file [/home/my-os-user/.local/share/RMD/settings.json].
Loaded Source:  MyRedditUser Saved

 Traceback (most recent call last):
  File "/usr/lib/python3.5/runpy.py", line 184, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib/python3.5/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "redditdownloader/__main__.py", line 133, in <module>
    run()
  File "redditdownloader/__main__.py", line 129, in run
    ui.display()
  File "redditdownloader/interfaces/terminal.py", line 17, in display
    self.print_info(prog)
  File "redditdownloader/interfaces/terminal.py", line 25, in print_info
    print('\n'*10, colorama.ansi.clear_screen())
  File "/home/mike/reddit-media-downloader/lib/python3.5/site-packages/colorama/ansitowin32.py", line 41, in write
    self.__convertor.write(text)
  File "/home/mike/reddit-media-downloader/lib/python3.5/site-packages/colorama/ansitowin32.py", line 162, in write
    self.write_and_convert(text)
  File "/home/mike/reddit-media-downloader/lib/python3.5/site-packages/colorama/ansitowin32.py", line 188, in write_and_convert
    self.convert_ansi(*match.groups())
  File "/home/mike/reddit-media-downloader/lib/python3.5/site-packages/colorama/ansitowin32.py", line 202, in convert_ansi
    self.call_win32(command, params)
  File "/home/mike/reddit-media-downloader/lib/python3.5/site-packages/colorama/ansitowin32.py", line 233, in call_win32
    winterm.erase_screen(params[0], on_stderr=self.on_stderr)
AttributeError: 'NoneType' object has no attribute 'erase_screen'
```

### Additional context
Setting `console_clear_screen": false` works around the issue.

**Comment by shadowmoose** [comment]
Hmm, interesting. This is a bug with the library RMD uses to clear the screen, as it fails to find a valid "erase terminal screen" code. 
I wasn't able to find this issue in my testing, but I'll use this report to try and wrap the error location now. Thanks for including the stack trace.

For those interested, RMD has a setting that should work for this contingency. Changing the setting ```threading.console_clear_screen``` to false should disable this character code, and will hopefully at least enable RMD to run.

**Comment by shadowmoose** [comment]
Should be fixed, pending release, as of ae06498ddae960e4514cc4172fdc11122a9c2118.

**Comment by shadowmoose** [comment]
Resolved as of [Release 3.0.0](https://github.com/shadowmoose/RedditDownloader/releases/tag/3.0.0). Feel free to reopen if this still needs more attention.
